### PR TITLE
[ MOD ]Grow some seeds

### PR DESCRIPTION
**Pull request by Honcharuk**
Allows you to grow some seeds in flower pots.

### How to use

Initially, we create a special pot for plants, with supports for them. Because I did not find the way to keep track of which pot the plant was planted on.
After a while, the plants turn into mature ones and need to be activated to prepare for harvesting.
Then you can disassemble the pot having received the harvest and the pot.

#### Summary

Summary: Content "Adds new mod Growable pots"

#### Additional context

The second attempt, reduced the number of recipes, you can add a system of fertilizers during planting for accelerated growth or increased yield.
If it will be necessary. Also, until all vegetables can be grown, you need to balance the time of ripening, I will be glad to receive your comments or suggestions.
Thank!

### CODE REVIEW

**Comment by Honcharuk** [comment]
default

**Comment by Honcharuk** [comment]
> 
> 
> Sounds interesting, but can you take a screenshot in English as well?

That's how we get the harvest
default

**Comment by Honcharuk** [comment]
> 
> 
> Frankly, I wouldn't mind something like that in vanilla. Indoor farming, however less effective it'd be than regular without specialized equipment or just growing some flowers in a pot to make the place more comfortable seems like a nifty addition.

It may be possible to do something like boxes for growing in a car as a part.
default

**Comment by AMurkin** [comment]
Summary line should be like:
Summary: Content "Adds new mod Growable pots"

Pros and Cons: It allows you to grow things under any condition.

**Comment by Honcharuk** [comment]
> 
> 
> Summary line should be like:
> Summary: Content "Adds new mod Growable pots"
> 
> Pros and Cons: It allows you to grow things under any condition.

Thanks, corrected the description.

**Comment by AMurkin** [comment]
Jenkins rebuild

File: data/mods/Growable_pots/items.json [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+    "cutting": 0,
+    "to_hit": 0,
+    "category": "tools"
+  },
+  {
+    "type": "GENERIC",
+    "id": "gp_cucumber",
+    "name": "Garden pot (growing cucumber)",
+    "name_plural": "Garden pot (growing cucumber)",
+    "description": "This is a garden pot growing cucumber. When ripe you need to activate to prepare for harvest.",
```
Comment (ZhilkinSerg) [inline data/mods/Growable_pots/items.json]:
```suggestion
    "description": "This is a garden pot growing cucumber.  When ripe you need to activate to prepare for harvest.",
```

File: data/mods/Growable_pots/items.json [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+      "moves": 0,
+      "target": "gp_cucumber_ready"
+    }
+  },
+  {
+    "type": "GENERIC",
+    "id": "gp_cucumber_ready",
+    "name": "Garden pot (ripe cucumber)",
+    "name_plural": "Garden pots (ripe cucumber)",
+    "description": "This is a garden pot containing a ready to harvest cucumber. Disassemble for harvest.",
```
Comment (ZhilkinSerg) [inline data/mods/Growable_pots/items.json]:
```suggestion
    "description": "This is a garden pot containing a ready to harvest cucumber.  Disassemble for harvest.",
```

File: data/mods/Growable_pots/items.json [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+    "cutting": 0,
+    "to_hit": 0,
+    "category": "tools"
+  },
+  {
+    "type": "GENERIC",
+    "id": "gp_garlic",
+    "name": "Garden pot (growing garlic)",
+    "name_plural": "Garden pot (growing garlic)",
+    "description": "This is a garden pot growing garlic. When ripe you need to activate to prepare for harvest.",
```
Comment (ZhilkinSerg) [inline data/mods/Growable_pots/items.json]:
```suggestion
    "description": "This is a garden pot growing garlic.  When ripe you need to activate to prepare for harvest.",
```

File: data/mods/Growable_pots/items.json [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+      "moves": 0,
+      "target": "gp_cucumber_ready"
+    }
+  },
+  {
+    "type": "GENERIC",
+    "id": "gp_garlic_ready",
+    "name": "Garden pot (ripe garlic)",
+    "name_plural": "Garden pots (ripe garlic)",
+    "description": "This is a garden pot containing a ready to harvest garlic. Disassemble for harvest.",
```
Comment (ZhilkinSerg) [inline data/mods/Growable_pots/items.json]:
```suggestion
    "description": "This is a garden pot containing a ready to harvest garlic.  Disassemble for harvest.",
```

**Pull request merged**